include/gz/sim/components/Model.hh

```diff
@@ -103,12 +104,30 @@ namespace serializers
         return _in;
       }
 
-      // Its super expensive to create an SDFElement for some reason
+      // Its super expensive to create an sdf::SDFPtr object.
+      // Workaround this by making it a static object so we only initialize it
+      // once.
+      // https://github.com/gazebosim/sdformat/issues/1478
+      sdf::Errors errors;
+      static sdf::SDFPtr sdfParsed;
```

Comment:
```suggestion
      static sdf::SDFPtr sdfParsed;
```

So there are definitely places where this seems to introduce regressions:

https://build.osrfoundation.org/job/gz_sim-ci-pr_any-homebrew-amd64/2111/consoleFull#-140582033379332c8d-713a-4d09-92ce-eb424edb41b6

<img width="3239" height="1245" alt="image" src="https://github.com/user-attachments/assets/4908d61d-7f8c-4d1b-be90-6e6b004db357" />

These tests will need to be fixed before we merge.